src/gsy_e/gsy_e_core/simulation/simulation.py

```diff
@@ -542,7 +541,10 @@ def _cycle_markets(self, slot_no: int) -> None:
             self._get_current_market_time_slot(slot_no), area=self.area
         )
 
-        self.area.cycle_coefficients_trading(self.progress_info.current_slot_time)
+        self.area.cycle_coefficients_trading(
+            self.progress_info.current_slot_time,
+            global_objects.profiles_handler.current_scm_profiles,
```

Comment:
@hannesdiedrich this is where the fee profiles are passed to the areas. Related to this comment:
https://github.com/gridsingularity/gsy-e/pull/1803/files#r1802887433